nixos/modules/services/web-apps/pgpkeyserver-lite.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pgpkeyserver-lite;
+  sksCfg = config.services.sks;
+
+  webPkg = cfg.package;
+
+in
+
+{
+
+  options = {
+
+    services.pgpkeyserver-lite = {
+
+      enable = mkEnableOption "pgpkeyserver-lite";
+
+      package = mkOption {
+        default = pkgs.pgpkeyserver-lite;
+        defaultText = "pkgs.pgpkeyserver-lite";
+        type = types.package;
+        description = "
+          Which webgui derivation to use.
+        ";
+      };
+
+      hostname = mkOption {
+        type = types.str;
+        description = "
+          hostname for webgui.
+        ";
+      };     
+
+      hkp_address = mkOption {
+        default = sksCfg.hkp_address;
+        type = types.listOf types.str;
+        description = "
+          pgpkeyserver-lite hkp listen address
+        ";
+      };
+
+      hkp_port = mkOption {
+        default = sksCfg.hkp_port;
+        type = types.int;
+        description = "
+          pgpkeyserver-lite hkp listen port
+        ";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+
+    services.nginx.enable = true;
+
+    services.nginx.virtualHosts = let
+      hkp_address = builtins.concatStringsSep " " cfg.hkp_address;
+      hkp_port = builtins.toString cfg.hkp_port;
+    in {
+      "${cfg.hostname}" = {
+        root = webPkg;
+        locations = {
+          "~ (.git|LICENSE|README.md)".extraConfig = ''
```

Comment:
I'd prefer simply removing these files while building the package or moving the public accessible files into a subfolder.